### ISSUE TITLE
Improve handling of ');' after comment in SA1009 code fix

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#2473]
**Input:**

```csharp
Console.WriteLine(string.Empty // Comment
    );
```

**Expected result:**

SA1009 is reported. The code fix produces the following:

```csharp
Console.WriteLine(string.Empty); // Comment
```

**Actual result:**

The code fix produces the following:

```csharp
Console.WriteLine(string.Empty) // Comment
    ;
```

⚠️ Make sure to not change the behavior when the `)` is followed by a member access operator.
